File: trunk/proj/Android/src/com/jasonzqshen/familyaccounting/core/masterdata/VendorMasterData.java
package com.jasonzqshen.familyaccounting.core.masterdata;

import com.jasonzqshen.familyaccounting.core.CoreDriver;
import com.jasonzqshen.familyaccounting.core.exception.NullValueNotAcceptable;

public class VendorMasterData extends MasterDataBase {
	public static final String FILE_NAME = "vendor.xml";

	/**
	 * 
	 * @param id
	 * @param descp
	 * @param parser
	 * @throws NullValueNotAcceptable 
	 */
	public VendorMasterData(CoreDriver coreDriver, MasterDataIdentity id,
			String descp) throws NullValueNotAcceptable {
		super(coreDriver, id, descp);
	}
}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/MasterData/VendorMasterData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.MasterData
{
    public class VendorMasterData : MasterDataBase
    {
        public static readonly String FILE_NAME = "vendor.xml";

        /// <summary>
        /// Construct
        /// </summary>
        /// <param name="coreDriver"></param>
        /// <param name="management"></param>
        /// <param name="id"></param>
        /// <param name="descp"></param>
        /// <exception cref="ArgumentNullException">argument is null</exception>
        public VendorMasterData(CoreDriver coreDriver,
                MasterDataManagement management, MasterDataIdentity id, String descp)
            : base(coreDriver, management, id, descp)
        {
        }
    }
}
